source/common/protobuf/utility.h

```diff
@@ -437,6 +437,13 @@ class ValueUtil {
    */
   static ProtobufWkt::Value boolValue(bool b);
 
+  /**
+   * Wrap double into ProtobufWkt::Value string value.
+   * @param d double to be wrapped.
+   * @return wrapped double.
+   */
+  static ProtobufWkt::Value numberValue(const double str);
+
```

Comment:
There's already a `numberValue(..)` utility function on line 459.
You can remove this function and its definition.

Reply:
removed.